Automattic/WP-Job-Manager
# Stop using pathinfo

### Overview
It has been reported that the usage of `pathinfo` in job manager can cause issues with resized images. The problem seems to be that `pathinfo` is locale aware which means that if there is a different server configuration, the filename won't be parsed correctly causing the image url to be malformed.

### Changes proposed in this Pull Request

* Switch to using `wp_check_filetype` instead.

### Testing instructions

* The underlying issue isn't reproducible on a local environment. To test this we just need to ensure that no new bug is introduced.
* Upload an image as a company logo or as a resume photo.
* Observe the resumes page or the jobs page and ensures that the thumbnail are displayed correctly.
